.github/workflows/image_build_and_test.yaml

```diff
@@ -150,7 +136,10 @@ jobs:
           ES_SPARK_30_2_12: elasticsearch-spark-30_2.12
           ES_SPARK_20_2_11: elasticsearch-spark-20_2.11
         # Skip SSL tests marked with `@pytest.mark.ssl`
-        run: python3.9 -m poetry run pytest -v tests/dataframe_tests -m "not ssl" --cov=tube --cov-append --cov-report term-missing --cov-report xml
+        run: | 
+          python3.9 -m poetry run pytest -v tests/dataframe_tests/test_dataframe_aggregation.py -m "not ssl" --cov=tube --cov-append --cov-report term-missing --cov-report xml
+          python3.9 -m poetry run pytest -v tests/dataframe_tests/test_dataframe_injection.py -m "not ssl" --cov=tube --cov-append --cov-report term-missing --cov-report xml
+          python3.9 -m poetry run pytest -v tests/dataframe_tests/test_parser.py -m "not ssl" --cov=tube --cov-append --cov-report term-missing --cov-report xml
```

Comment:
this is a bit dangerous, if we add new files we might not remember to add them here 🤔 

maybe we could use pytest markers instead - i think pytest's `-m` option supports and/or, so maybe something like this?
```
python3.9 -m poetry run pytest -v tests/dataframe_tests -m "schema_ibdgc and not ssl" [...]
python3.9 -m poetry run pytest -v tests/dataframe_tests -m "schema_midrc and not ssl" [...]

and at the end:
python3.9 -m poetry run pytest -v tests/dataframe_tests -m "not ssl and not schema_ibdgc and not schema_midrc" [...]
```
and mark each test with `@pytest.mark.schema_ibdgc` or `@pytest.mark.schema_midrc`
until we figure out a way to reset the initialized dictionary 😕

Reply:
@thanh-nguyen-dang  we also need `python3.9 -m poetry run pytest -v tests/dataframe_tests -m "not ssl and not schema_ibdgc and not schema_midrc" [...]` for potential tests that don't have a schema marker